static/css/main-bootstrap.css

```diff
@@ -177,6 +177,13 @@
 .glyphicon-world:before {
    content: "\E372";
 }
+.glyphicon-book-open:before {
+   content: "\E352";
+   margin-left: 5px;
+   margin-right: 5px;
```

Comment:
I think it would look nicer to have just a margin-left: 8px (or so) and delete the margin-right.